src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private String color;
+
+    public Circle(double radius, String color) {
+        setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, radius: "
+                + radius + " units, color: " + getColor());
```

Comment:
Use 'name()' for getting String representation of enum constants. If 'color' is an enum type, you should use 'color.name()' instead of 'getColor()' for the output.